### PR TITLE
Calculator wireframe

### PR DESCRIPTION
**Pull request by damdafayton**
- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, create a new Calculator.js file.
- In Calculator.js, create a React component.
- Don't add any extra design.
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use class-based components.
- Once the Calculator is ready, import it and use it in your main component (App.js).

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,16 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <header className="" />
+      <body>
+        <Calculator />
+      </body>
```
Comment (mikethreels) [inline src/App.js]: - [x] the `App` component should be created as a class-based component according to the requirements.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import Cell from './Cell';
+
+export default class Calculator extends React.Component {
+  render() {
+    return (
+      <div className="">
+        <div className="text-white text-end">
+          <Cell myClass="cell bg-dark fs-3">0</Cell>
+        </div>
+        <div className="row row-cols-4">
```
Comment (mikethreels) [inline src/components/Calculator.js]: - [x] in order to fix the linter error you asked about you can fix it by including a `constructor` method, similar to below:
```
  constructor(props) {
    super(props);
    this.state = {};
  }
```

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Yucel, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 2 above)
Comment (damdafayton) [reply src/components/Calculator.js]: @mikethreels Thanks

**Review by Meltrust — APPROVED** [review]
Hi @damdafayton,


## Wow, you did it 🎉
Brilliant
## Thank you for changing your App component to class-based, as requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**